pkg/cmd/stack/deploy.go

```diff
@@ -229,8 +229,11 @@ func deployVolume(ctx context.Context, volumeName string, s *model.Stack, c *kub
 		if old.Labels[okLabels.StackNameLabel] != s.Name {
 			return fmt.Errorf("name collision: the volume '%s' belongs to the stack '%s'", pvc.Name, old.Labels[okLabels.StackNameLabel])
 		}
-		if err := volumes.Update(ctx, &pvc, c); err != nil {
-			return fmt.Errorf("error updating volume of service '%s': %s", pvc.Name, err.Error())
+		if !volumes.IsEqual(&pvc, old) {
```

Comment:
Can we do this in every case? if they are equal, it should do nothing.
Also, can you try that it is possible to change the volume size and the change takes effect in the volume?